oidc-controller/api/core/config.py

```diff
@@ -1,12 +1,98 @@
+import json
 import logging
+import sys
+import logging.config
+import structlog
 import os
 from enum import Enum
 from functools import lru_cache
 from typing import Optional, Union
 
 from pydantic import BaseSettings
 
-logger = logging.getLogger(__name__)
+from pathlib import Path
+
+logging.basicConfig(
+    format="%(message)s",
+    stream=sys.stdout,
+    level=logging.DEBUG,
+)
+
+# Use environment variable to determin logging format
+# fallback to logconf.json
+# finally default to true
+use_json_logs: bool
+
+time_stamp_format = "%Y-%m-%d %H:%M.%S"
+with open((Path(__file__).parent.parent / "logconf.json").resolve()) as user_file:
```

Comment:
It would be nice to allow the path to the configuration file to be overridden, and use this default file if nothing is specified.

The main reason is that `logconf.json` is placed with the source code and mounting a file/folder at this location might result complicated, while referencing a file located elsewhere in the filesystem will be easier for users to manage.

Reply:
Setting the logconf.json file should probably be exposed via ENVVAR, and as @esune noted, default to `logconf.json`